test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
 initialization =
-    import os
-    import subprocess
-    os.chdir("${buildout:directory}")
-    pkgdir = subprocess.Popen(
-        '${buildout:directory}/bin/package-directory relative',
-        shell=True, stdout=subprocess.PIPE).stdout.read().strip()
-    if '--absolute' in sys.argv: pkgdir = os.path.abspath(pkgdir) ; sys.argv.remove('--absolute')
```

Comment:
For pycodestyle I'm actually still using this within emacs: the goal is that all paths are absolute, so that emacs can link the output.
Would it be possible to still support it? 😳